package.json

```diff
@@ -92,6 +95,8 @@
     "core-js-builder": "~3.6.5",
     "css": "^3.0.0",
     "dts-bundle-generator": "~5.3.0",
+    "eslint": "^8.13.0",
+    "eslint-config-prettier": "^8.5.0",
```

Comment:
IIRC, this has to be added to the `.eslintrc.js` file to turn those rules off that conflict with Prettier.  It's probably a no-op right now, since `no-unused-vars` doesn't conflict with Prettier.

Specifically, I think we need to do something [like this](https://github.com/prettier/eslint-config-prettier#installation) in `.eslintrc.js`:
```diff
module.exports = {
  root: true,
  parser: '@typescript-eslint/parser',
  plugins: [
    '@typescript-eslint',
  ], 
+ extends: {
+   // prettier must be last in this list to ensure that it's not overridden by other configs
+   prettier,
+ }
  rules: {
    "@typescript-eslint/no-unused-vars": [2]
  }
};
```

(FWIW, I'm a big fan of commenting the heck out of lint config files to try to make the 'whys' ridiculously apparent)

Reply:
yeah, I think you're right that it's a good idea to set that up now. When we have no rules set up it doesn't make much difference, but why not make sure it's set up from the beginning!